pymongo/ocsp_support.py

```diff
@@ -132,6 +148,10 @@ def _verify_signature(
             key.verify(signature, data, algorithm)  # type: ignore[arg-type]
         elif isinstance(key, _EllipticCurvePublicKey):
             key.verify(signature, data, _ECDSA(algorithm))  # type: ignore[arg-type]
+        elif isinstance(key, _X25519PublicKey) or isinstance(
```

Comment:
You can pass a tuple with both classes to combine these two isinstance calls into one.